tests/admin_filters/tests.py

```diff
@@ -43,6 +43,22 @@ def queryset(self, request, queryset):
             return queryset.filter(year__gte=2000, year__lte=2009)
 
 
+class NotNinetiesListFilter(SimpleListFilter):
```

Comment:
I think I'd define these in the test since they're only used once. That makes it easier to see where it's used.